ajax.php

```diff
@@ -34,7 +34,7 @@
      * Type: HTTP POST
      *
      * Parameters:
-     * 	- content_id
+     *     - content_id
```

Comment:
This seems weird. Is the indentation deliberately different from the other items in the list?